infra/.ci-env

```diff
@@ -0,0 +1,18 @@
+BINANCE_MNEMONIC="clutch captain shoe salt awake harvest setup primary inmate ugly among become"
+BINANCE_ADDRESS=0xa0df350d2637096571F7A701CBc1C5fdE30dF76A
+BITCOIN_PK=cUJCHRMSUwkcofsHjFWBELT3yEAejokdKhyTNv3DScodYWzztBae
+BITCOIN_ADDRESS=mwjUmhAW68zCtgZpW5b1xD5g7MZew6xPV4
+BITCOINCASH_PK=cSEohZFQLKuemNeBVrzwxniouUJJxdcx7Tm6HpspYuxraVjytieW
+BITCOINCASH_ADDRESS=bchreg:qp6tejc0ghtjeejcxa97amzvxvzacjt4qczpy2n3gf
+DIGIBYTE_PK=efbJxdzwR1tZD7KWYmKBhmxR6TNb25P9z29ajaoALhkn1LdNe7Ci
+DIGIBYTE_ADDRESS=shb4rf33ozMX8rinHsC6GghrvvA8RKTyGb
+DOGECOIN_PK=cRZnRgH2ztcJupCzkWbq2mjiT8PSFAmtYRYb1phg1vSRRcNBX4w4
+DOGECOIN_ADDRESS=n3PSSpR4zqUKWH4tcRjP9aTwJ4GmixQXmt
+ETHEREUM_MNEMONIC="clutch captain shoe salt awake harvest setup primary inmate ugly among become"
+ETHEREUM_ADDRESS=0xa0df350d2637096571F7A701CBc1C5fdE30dF76A
+FILECOIN_PK=7b2254797065223a22736563703235366b31222c22507269766174654b6579223a22756d6a634e436a487a5438455757485849754a4c4b58745035437153323435666238626c656c756e5448493d227d
+FILECOIN_ADDRESS=t1ej2tountzqwnu6uswhqdzvw6yy5xvcig6rxl2qa
+TERRA_PK=a96e62ed3955e65be32703f12d87b6b5cf26039ecfa948dc5107a495418e5330
+TERRA_ADDRESS=terra10s4mg25tu6termrk8egltfyme4q7sg3hl8s38u
+ZCASH_PK=cNSVbbsAcBQ6BAmMr6yH6DLWr7QTDptHwdzpy4GYxGDkNZeKnczK
+ZCASH_ADDRESS=tmCTReBSJEDMWfFCkXXPMSB3EfuPg6SE9dw
```

Comment:
What is the purpose of this file as oppose to the usual `.env`?

Reply:
The `.env` is for setting those environment variables for all processes. It has a format like this:
```
export BITCOIN_PK=cUJCHR...
```

In order to pass environment variables as a file to the `docker run` command using the `--env-file` option, the above format was the required one.

This shouldn't be required anymore, if `docker-compose` is used to `build` and `up` the containers. 👍

Reply:
This file has now been removed!